main.private-endpoints.databricks.tf

```
@@ -48,8 +48,8 @@ resource "azurerm_private_endpoint" "databricks" {
   resource_group_name = azurerm_resource_group.this.name
   subnet_id           = module.gh_runner_vnet.subnets.pe_subnet.resource_id
 
-  tags = merge(var.tags, {
-    Description = "Private endpoint for Azure Databricks '${each.value.dbx_details.resource_name}' located in resource group '${each.value.dbx_details.resource_group_name}'. Part of the '${var.system_name}' infrastructure for GitHub hosted Actions runners"
+  tags = merge(var.tags, each.value.tags, {
+    Description = "PE for Azure Databricks '${each.value.dbx_details.resource_name}' in resource group '${each.value.dbx_details.resource_group_name}'. Part of the '${var.system_name}' infrastructure for GitHub hosted Actions runners"
   })
```

you need to change the order of this merge, the last input to the function wins.

the functionality we want is that `Description` tag from code is default/fallback. Then `Description` can be overridden by  both `var.tags` (general for all resources) and from `each.value.tags` (individually for each resource).


```suggestion
  tags = merge({
    Description = "PE for Azure Databricks '${each.value.dbx_details.resource_name}' in resource group '${each.value.dbx_details.resource_group_name}'. Part of the '${var.system_name}' infrastructure for GitHub hosted Actions runners"
    },
    var.tags,
    each.value.tags,
  )
```